app/views/general/_opengraph_tags.html.erb

```diff
@@ -9,7 +9,7 @@
 <%= content_for :open_graph_tags %>
 
 <meta property="og:site_name" content="<%= site_name %>" />
-<meta property="og:url" content="<%= request.original_url %>" />
+<meta property="og:url" content="<%= request.original_url.encode("utf-8", invalid: :replace, undef: :replace, replace: "_") %>" />
```

Comment:
`force_encoding('utf-8')` should work, I'll give it a try (and add some more info)